### ISSUE TITLE
Authentication Error. Password Grant Type

### ISSUE THREAD

**Issue by insanerwayner** [KraigM/homebridge-wink#91]
```
[11/18/2017, 4:28:21 AM] [Wink] There was a problem authenticating with Wink: This client is not permitted to use the password grant type
HAP Warning: Characteristic 00000054-0000-1000-8000-0026BB765291 not in required or optional characteristics for service 0000003E-0000-1000-8000-0026BB765291. Adding anyway.
HAP Warning: Characteristic 00000054-0000-1000-8000-0026BB765291 not in required or optional characteristics for service 0000003E-0000-1000-8000-0026BB765291. Adding anyway.
```

**Comment by andregaulin** [comment]
I'm seeing this error as well. It seems to have started after I updated the firmware on my hub to the latest version that was pushed a little over 10 days ago.

**Comment by gleep52** [comment]
I started getting this error recently too though I don't know when it started happening.  I get it with the android API key and my own developer key...

**Comment by psolin** [comment]
Yes, this is now failing constantly, making this plugin unusable.  Tried using a more public key, as well as my own private one, and nothing works.

**Comment by gleep52** [comment]
I had to remove the username and password from my config file and now my private keys are working again.  Weird.  Must have been an update to homebridge-wink.

**Comment by coheredigital** [comment]
I removed my username and password as @gleep52 mentioned and still nothing. Any official solution or further recommendations?

**Comment by jeffl78** [comment]
I’m having the same problem, removing the password & username didn’t not help.  Can anyone help us with this?

**Comment by coheredigital** [comment]
Hey @jeffl78 check out the homebridge-wink3 package. It worked perfectly for me first try. Sorry I didn’t add this to my previous post. I don’t have a link but a quick search will get you the repo I’m sure. Just make sure you remove the previously installed wink package as they will attempts to register under the same name.

**Comment by jeffl78** [comment]
Thanks, I’ll give it a try with my first gen wink hub & let you know what happens.

**Comment by gleep52** [comment]
the wink3 package should work for you without the email address and password.  I believe that's what I'm using right now.

**Comment by jeffl78** [comment]
Wink3 works great, thanks!

**Comment by topgunpreet** [comment]
I tried both homebridge-wink and homebridge-wink3 and still have the same issue. I am running wink hub 1.

**Comment by gleep52** [comment]
do you have the key in the config file, or are the lines for email/password still there?  You'll need to remove those lines completely and just use the keys instead.

**Comment by topgunpreet** [comment]
Removed lines of email/password and it is working now with homebridge-wink3. Thank you. I assume wink3 is the correct package?

**Comment by gleep52** [comment]
Yes it appears wink3 no longer understands the email password lines and if you take them out, it works again.
